app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -215,6 +215,12 @@ class HomeFragment : Fragment() {
             sessionControlInteractor,
             homeViewModel
         )
+
+        // This has to be called separately from the consumeFrom block since the coroutine doesn't
```

Comment:
nit: this is unclear. Which consumeFrom block (see my comment below)? What coroutine? What do you mean by "delays our start up"? How? What init block? How does it force our UI to render?